pkg/generate/app/cmd/newapp_test.go

```diff
@@ -137,6 +176,7 @@ func TestValidate(t *testing.T) {
 			}
 		}
 	}
+	envFileOperation("/tmp/envfile123", "delete")
```

Comment:
should be in a defer before starting the loop to avoid the risk of leaving files around.

and the path should be based on ioutil.TempFile